scripts/devportal/update-themes-and-modules.sh

```diff
@@ -25,33 +30,53 @@ function add_all_repositories {
 function add_all_modules_repositories {
     for repo_name in ${modules[@]}
     do
-        git remote add -f ${repo_name} git@bitbucket.org:pronovix/${repo_name}.git
+      register_repository ${repo_name}
```

Comment:
If I'm not mistaken, you're assuming that the directory names in `web/modules/devportal/` are repo names – which is merely a coincidence. Could you pls also document this need?

Reply:
Yep, you are right, this requirement is coming from the original code. It was also weird for me, but with this approach at least we do not expose our repo names.

Reply:
This is already documented here: https://github.com/Pronovix/devportal-starterkit#updating-devportal-themes-and-modules